cmd/agent/Dockerfile

```diff
@@ -2,9 +2,9 @@ FROM alpine:latest as certs
 RUN apk add --update --no-cache ca-certificates
 
 FROM scratch
-
+ARG ARCH
```

Comment:
Could we not default this to `amd64` and avoid having to change so many places in the Makefile? We're only distributing Docker images for amd64 platforms.

Reply:
Sorry but I don't get it...I think currently we don't have any default of `amd64`?

Reply:
You're correct, but it's a defect, our build process is dependent on the architecture that Travis runs on, which is amd64. It's good that you're making it explicit by introducing the `ARCH` argument, however we can simply default that param to `amd64` (to match the new binary file names), and not pass it from the Makefile, since we only want amd64 Docker images.

Reply:
Thanks @yurishkuro for your patience and explanation! Yes I agree with your point, and I have changed the default value of `ARCH` param to `amd64`. 

However I would like to keep the `--build-arg` option in the `Makefile` so that non-amd64 users, including me (arm64 user) and those s390x users can still build docker images on our machine with a simple `make docker` command. Would this ok to you? If not I will change to satisfy what the community needs. Thanks again!

Reply:
fair enough

Reply:
Thanks!